### DISCUSSION TITLE
About errors using Analytics. Next.js(/app)

### DISCUSSION THREAD

**Discussion by snowriceDD** [vercel/next.js#51343] [Help] [answered]
### Summary

<img width="670" alt="image" src="https://github.com/vercel/next.js/assets/103573547/5c321bc9-a9cb-4b28-98c1-b23f6fe160b5">

I'm using Vercel Analytics, and I'm getting these build errors.

<img width="502" alt="image" src="https://github.com/vercel/next.js/assets/103573547/b52a4a00-770e-4b31-aa48-7eefd1c846f0">

Shouldn't I use the Analytics component in this way?
I'm a beginner and need help. thx.

**Comment by SamKomesarook** [comment, marked as answer]
How did you install the package? Yarn or npm? What was the package name you installed?

**Reply by snowriceDD** [reply]
I installed it using yarn add @vercel/analytics.

The answer to this problem was in the package.json.
A missing dependency was causing the error, and when I wrote an arbitrary dependency, the deployment worked fine.

Thank you for thinking about this with us!
